www/~/%username/emails/verify.html.spt

```diff
@@ -4,14 +4,17 @@ from datetime import timedelta
 
 from aspen import Response
 from aspen.utils import utcnow
-from gratipay.utils import emails, get_participant
+from gratipay.utils import emails, get_participant, b64decode_s
 
 [-----------------------------------------------------------------------------]
 
 participant = get_participant(state, restrict=False)
 banner = '~' + participant.username
 if participant == user.participant:
-    email = request.qs.get('email', '')
+    if 'email' in request.qs:
+        email = request.qs['email']
+    else:
+        email = b64decode_s(request.qs.get('email64', ''), default='')
```

Comment:
Okay, so since verification emails [expire](https://github.com/gratipay/gratipay.com/blob/0ee9aa51064290ce6acbbfca0df23a8a81b1419f/gratipay/models/participant.py#L59) after 24 hours, we could drop `email` and rename `email64` to `email` after this has been deployed for 24 hours, yes?